### PR TITLE
v21: copy v20 kvm

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: service/controller/kvm/v21/error.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package v20
```
Comment (sslavic) [inline service/controller/kvm/v21/error.go]:
```suggestion
package v21
```

File: service/controller/kvm/v21/resource_set.go [hunk 2]
```diff
@@ -0,0 +1,436 @@
+package v20
```
Comment (sslavic) [inline service/controller/kvm/v21/resource_set.go]:
```suggestion
package v21
```

File: service/controller/kvm/v21/version_bundle.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package v20
```
Comment (sslavic) [inline service/controller/kvm/v21/version_bundle.go]:
```suggestion
package v21
```

**Comment by rossf7** [comment]
Last copy PR will merge once the build is green. https://github.com/giantswarm/giantswarm/blob/master/processes/merging-without-approval.md

**Pull request merged**